blocks/header/header.css

```diff
@@ -476,6 +482,367 @@ header .megamenu>.section-outer {
+          top: 50%;
+          transform: translate(0, -50%) scale(0.45) rotate(90deg);
+          right: 10px;
+        }
+
+        .accordion details[open] summary::after {
+          transform: translate(0, -50%) scale(0.45) rotate(-90deg);
+        }
+
+        .accordion details[open] .accordion-item-body {
+          transition: none;
+        }
+
+        .accordion .accordion-item-body {
+          .button-container {
+            margin: 0;
+
+            &>* {
+              width: 100%;
+              display: block;
+
+              .button {
+                width: 100%;
+              }
+            }
+          }
+
+          .sub-content {
+            padding: 20px;
+            border-left: 3px solid var(--berry);
+
+            .icon {
+              height: 30px;
+              padding-right: 10px;
+            }
+
+            a {
+              display: flex;
+              color: inherit;
+              text-decoration: underline;
+              padding-left: 20%;
+
+              &::before {
+                content: url("/icons/icon-arrow-right.svg");
+                position: relative;
+                display: block;
+                height: 20px;
+                padding-right: 10px;
+              }
+            }
+          }
+        }
+
+      }
+
+      .section>* {
+        width: 95% !important;
+      }
+    }
+
+    .columns-container .columns-wrapper {
+      width: 100%;
+
+      .columns.block,
+      .columns-2-cols {
+        max-width: 100% !important;
+      }
+    }
+
+    .columns-container .columns.block>div {
+      display: flex;
+      flex-direction: column;
+      justify-content: flex-start;
+      align-items: flex-start;
+      gap: 15px;
+      width: 100%;
+
+      >div {
+        width: 100%;
+        padding: 15px;
+      }
+
+      ul {
+        border: none !important;
+      }
+    }
+  }
+
+
+  .nav-sections .megamenu-container {
+    display: none;
+  }
+
+  .nav-mobile-header {
+    position: relative;
+    display: block;
+    width: 100%;
+    background-color: var(--light-gray);
+
+    & .main-menu-section {
+      display: none;
+
+      & h2 {
+        margin-left: 20px;
+        font-weight: var(--font-weight-bold);
+      }
+    }
+
+    & .back-button {
+      position: relative;
+      font-weight: var(--font-weight-normal);
+
+      &::before {
+        content: url("/icons/chevron-left-solid.svg");
+        padding-right: 5px;
+        height: 7px;
+        width: 12px;
+      }
+    }
+  }
+
+  .nav-brand,
+  .nav-hamburger {
+    display: none;
+  }
+
+  ul.primary-nav-items {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    align-items: flex-start;
+    gap: 30px;
+    padding: 0;
+    margin: 0;
+    width: 100%;
+
+    li {
+      gap: 20px;
+      width: 100%;
+    }
+  }
+
+  .nav-sections ul.primary-nav-items>li:not(.button-container)>a:not(.button) {
+    width: 90%;
+    padding-bottom: 15%;
```

Comment:
Don't use a % here. You can see why not if you view this in 1000px wide viewport.